apps/etterlatte-behandling/src/main/kotlin/BeanFactory.kt

```diff
@@ -192,8 +192,8 @@ abstract class CommonFactory : BeanFactory {
 
 class EnvBasedBeanFactory(val env: Map<String, String>) : CommonFactory() {
     private val logger = LoggerFactory.getLogger(this::class.java)
-    private val datasourceBuilder: DataSourceBuilder by lazy { DataSourceBuilder(env) }
-    override fun datasourceBuilder() = datasourceBuilder
+
+    override fun dataSource() = DataSourceBuilder.createDataSource(env, maxPoolSize = 3)
```

Comment:
Behandling har 11 aktive connections nå, så 3 er vel litt for lite?

Reply:
Enig den, var for å se hva som måtte til for å få til å starte appen 😛